tests/integration/test_basic.py

```diff
@@ -32,12 +32,12 @@ def test_add_and_remove_unit(juju: jubilant.Juju):
     juju.wait(lambda status: jubilant.all_active(status) and len(status.apps[charm].units) == 1)
 
 
-# Tests config get, config set, run, and exec
+# Tests config get, config set, trust, run, exec, and cli with input
```

Comment:
I realise this is getting a bit out of hand. I've opened https://github.com/canonical/jubilant/issues/84 to track: split integration tests into multiple files with a module-scoped fixture.